array/longest_non_repeat.py

```diff
@@ -71,6 +71,7 @@ def test_longest_non_repeat_two(self):
         string = "pwwkew"
         self.assertEqual(longest_non_repeat_two(string), 3)
         
+
 if __name__ == "__main__":
     
```

Comment:
The empty line can be deleted.

Reply:
Oh, my IDE puts it there automatically according to the PEP guide.